docs/data/base/components/portal/portal.md

```diff
@@ -8,18 +8,24 @@ packageName: '@mui/base'
 
 # Portal
 
-<p class="description">The portal component renders its children into a new "subtree" outside of the current DOM hierarchy.</p>
+<p class="description">The <code>Portal</code> component lets you render its children into a DOM node that exists outside of its own DOM hierarchy.</p>
 
-The children of the portal component will be appended to the `container` specified.
-The component is used internally by the [`Modal`](/material-ui/react-modal/) and [`Popper`](/material-ui/react-popper/) components.
+`Portal` is a utility component built around [React's `createPortal()` API](https://reactjs.org/docs/portals.html).
+It gives you the functionality of `createPortal()` in a convenient component form.
 
-{{"component": "modules/components/ComponentLinkHeader.js", "design": false}}
+> 💡 According to [the React docs](https://reactjs.org/docs/portals.html), portals are useful when "you need the child element to visually 'break out' of its container."
 
-## Example
+The `Portal` component is used internally by Material UI's [`Modal`](/components/modal/) and [`Popper`](/components/popper/) components.
 
-{{"demo": "SimplePortal.js"}}
+## Basic usage
+
+Normally, child elements are appended to the nearest parent node.
```

Comment:
IMO something along these lines would read better:

```suggestion
Normally, children of a component are rendered within this component's DOM tree.
```

This is one of the places where having a diagram could make the concept much easier to understand.